### ISSUE TITLE
Toolbar collapse button is missing status indicator

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1636]
The state of the toolbar collapse button cannot be determined programmatically: it does not contain a value or indication whether it is open or closed
